src/routing/Router.tsx

```diff
@@ -1,38 +1,66 @@
 import React from 'react';
-import { BrowserRouter, Routes } from 'react-router-dom';
+import {
+  BrowserRouter,
+  createRoutesFromChildren,
+  RouteObject,
+  Routes,
+} from 'react-router-dom';
 import { Formik } from 'formik';
 import { RouterProps } from './types';
 
 import FormTitle from '../form-layout/FormTitle';
 import FormFooter from '../form-layout/FormFooter';
+import { RouterContext } from './RouterContext';
+
+const routeObjectsReducer = (routeObjectsArray: RouteObject[]) => {
```

Comment:
Could we add some ts docs to give a quick example of what the data object looks like? Or at least explain how what this function is returning.

Reply:
yes, oops forgot to build the docs